docs/learning/tutorials/develop-desktop-viewer.md

```diff
@@ -2,19 +2,15 @@
 
 ### Setup
 - [Install necessary prerequisites]($docs/getting-started/development-prerequisites)
-- [Clone imodeljs-samples repo](https://github.com/imodeljs/imodeljs-samples)
-- Have access to a cloud hosted iModel. If you do not have access to one, follow one of our tutorials to [create an online iModel]($docs/learning/tutorials/index.md)
+- [Clone desktop-starter repo](https://github.com/imodeljs/desktop-starter)
 
 ### Build
 - Open the cloned repo in VS Code
-- `cd interactive-app`
-- `cd simple-viewer-app`
-- Edit interactive-app\simple-viewer-app\\.env.local
-    > `imjs_test_imodel` = The name of your iModel<br/>
-
 - `npm install`
-- `npm run build`
 
 ### Run
-- `npm run electron`
+- `npm run start`
+
+The app will open a pre-packaged offline snapshot iModel. You will likely want to use your own snapshot iModel. Follow the [Create a snapshot iModel]($docs/learning/tutorials/create-test-imodel-offline) tutorial. Then open your briefcase by clicking the Home button and selecting "Open Offline Snapshot".

```

Comment:
I think in the new terms, this wouldn't technically be a briefcase.  A briefcase is an iModel that is synchronized with the Hub and not a snapshot iModel.

I'd suggest the last line to say, `Then open your Snapshot by clicking the Home button and selecting "Open Offline Snapshot"`.

I do want us to also change the name of that button to just say "Open Snapshot".

@kabentley is this the correct wording for the new changes to the handling of Briefcase?